scripts/visualization-data-converter.sh

```diff
@@ -1,4 +1,18 @@
-#!/bin/bash
```

Comment:
This file is referenced by `.github/workflows/benchmark_visualization.yml` - can you update it please?